### ISSUE TITLE
Warning notice when running `npm install --save lell`

### ISSUE THREAD

**Issue by joshuaquek** [arkverse/lell#3]
I got this output when I ran `npm install --save lell`

```bash
npm install --save lell                                                                                                                                                                 [18/06/13| 3:28PM]
+ lell@1.2.4
added 4 packages from 8 contributors and audited 4 packages in 6.466s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
```

When I ran `npm audit`, this was the output that I received:

```bash
npm audit                                                                                                                                                                               [18/06/13| 3:28PM]

                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ lell                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ lell > lodash                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 4 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

**Comment by joshuaquek** [comment]
Would there be any update to patch this?

**Comment by joshuaquek** [comment]
Hoping that this can be patched, because I really love this library. It has huge potential since its super easy to pick up and integrate into existing code.

**Comment by kidcosmic** [comment]
Published update 1.2.5 for you sir. Enjoy! :)
